src/store/sagas/ethereum/wallets.ts

```diff
@@ -0,0 +1,38 @@
+import { put, takeEvery } from 'redux-saga/effects';
+
+import * as ethereumActions from '../../actions/ethereum/wallets';
+import * as loaderActions from '../../actions/loader';
+
+async function fetchWallets(): Promise<any> {
+  const res = await fetch(
+    `http://localhost:5000/api/v1/ethereum/wallets?groupBy=from&limit=50`
```

Comment:
We need some sort of common helper function for determining the `http://localhost:5000` part based on NODE_ENV ('production' or 'dev'). For prod make it return whatever you want, we will fill that once we have deployed backend server. For dev return localhost:5000.

Reply:
Will make it in task with line chart